src/segment.go

```diff
@@ -112,16 +112,37 @@ func (segment *Segment) getValue(property Property, defaultValue string) string
 	return defaultValue
 }
 
-func (segment *Segment) shouldIgnoreFolder(cwd string) bool {
-	if value, ok := segment.Properties[IgnoreFolders]; ok {
+func (segment *Segment) shouldIncludeFolder(cwd string) bool {
```

Comment:
This function was switched to return the positive case instead of the negative, to make it easier to reason about.